packages/atlas/src/views/global/YppLandingView/YppAuthorizationModal/YppAuthorizationModal.tsx

```diff
@@ -145,18 +339,28 @@ export const YppAuthorizationModal: FC<YppAuthorizationModalProps> = ({ currentS
         return {
           title: 'Terms & conditions',
           // TODO: add proper copy once it's available in figma https://www.figma.com/file/oQqFqdAiPu16eeE2aA5AD5?node-id=1637:118716#267556722
-          description:
-            'Once automatic YouTube videos sync is available, in order for it to work, your Atlas channel [NEEDS TO DO WHAT?]. This is purely a technical measure and does not affect ownership and rights to the content uploaded to you Atlas channel.',
-          primaryButton: { text: 'Accept terms & sign' },
+          description: `Once automatic YouTube videos sync is available, in order for it to work, your ${APP_NAME} channel [NEEDS TO DO WHAT?]. This is purely a technical measure and does not affect ownership and rights to the content uploaded to you ${APP_NAME} channel.`,
+          primaryButton: {
+            text: 'Accept terms & sign',
+            onClick: handleAcceptTermsAndSubmit,
+          },
           additionalSubtitleNode: (
-            <AdditionalSubtitle>
-              <Text variant="h400" as="span">
+            <AdditionalSubtitleWrapper>
+              <Text variant={smMatch ? 'h400' : 'h300'} as="h3" margin={{ bottom: 4 }}>
+                Transaction fee
+              </Text>
+              <Text variant="t200" as="p" color="colorText" margin={{ bottom: 6 }}>
+                Applying for the program requires requires a blockchain transaction, which comes with a fee of{' '}
+                <NumberFormat as="span" variant="t200" value={updateChannelFee} withToken /> . Transaction fees are
```

Comment:
Unnecessary space before the dot

```suggestion
                <NumberFormat as="span" variant="t200" value={updateChannelFee} withToken />. Transaction fees are
```